### PR TITLE
sentry integration

### PR DESCRIPTION
**Pull request by labhvam5**

### CODE REVIEW

File: fyle_qbo_api/sentry.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        )
+
+    @staticmethod
+    def traces_sampler(sampling_context):
+        # avoiding ready APIs in performance tracing
+        if sampling_context.get('wsgi_environ') is not None:
+            if sampling_context['wsgi_environ']['PATH_INFO'] in ['/ready']:
+                return 0
+
+        return 1.0
```
Comment (ashwin1111) [inline fyle_qbo_api/sentry.py:28]: Can we reduce it to 0.5? If we are not getting much we can increase this.

Comment (ashwin1111) [reply fyle_qbo_api/sentry.py:28]: I assume this is for traces sample rate, correct me if I am wrong

**Review by ashwin1111 — APPROVED** [review]

**Comment by ashwin1111** [comment]
@labhvam5 make sure you do squash merge

File: fyle_qbo_api/sentry.py (hunk 1 above)
Comment (labhvam5) [reply fyle_qbo_api/sentry.py:28]: yes this is the traces sample rate.

File: fyle_qbo_api/sentry.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    def init():
+        sentry_sdk.init(
+            dsn=os.environ.get('SENTRY_DSN'),
+            send_default_pii=True,
+            integrations=[DjangoIntegration()],
+            environment='quickbooks-api',
+            traces_sampler=Sentry.traces_sampler,
+            release=os.environ.get('RELEASE'),
+            attach_stacktrace=True,
+            request_bodies= 'small',
```
Comment (ashwin1111) [inline fyle_qbo_api/sentry.py:19]: `request_bodies='small'`
No spacing

**Pull request merged**